docs/contributing/contributing-guidelines.txt

```diff
@@ -0,0 +1,35 @@
+.. _contributing-guildelines:
+
+***********************
+Contributing Guidelines
+***********************
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+
+Contributing Guidelines
+=======================
+
+When contributing code to Mongoid, the following guildelines should be followed:
+
+#. Create a fork of Mongoid.
+#. Create a new branch in that fork.
+#. Make your changes.
+#. Ensure that the proposed changes have adequate test coverage.
+#. Raise a PR against Mongoid master. If these changes correspond to a specific
+   Jira ticket, title the PR: "MONGOID-<ticket #> Description of Changes".
+#. The Mongoid team will review the PR and make comments/suggest changes.
+#. Once all of the changes and fixes are made, and the Mongoid team determine
+   the PR fit for merging, we will merge the PR into master and determine
+   whether it needs to be backported.
+#. Backports to 7.3-present are done if the change is a bug fix, is not backwards
```

Comment:
It would be ideal if this could be expressed on a relative basis, e.g. "Backports to the current and previous major version are done..." so that it doesn't need to be updated on every release.

If the policy is taking effect from version 7.3, you could prefix it with "Beginning from version 7.3..."

Reply:
I like this suggestion, but it's worth being specific for the time being. @Neilshweky I'd recommend changing the following bullet to 

```
#. Backports to previous stable versions are done if the change is a bug fix, is not backwards breaking, and if the commit is applicable to the corresponding stable branch. Presently backport candidates would include versions 7.3-8.0.
```

Reply:
done!

Reply:
A second thought here, this probably belongs in a completely separate policy like "Security & Patching Policy". Typically backports aren't covered in Contributing Guidelines (unless you are asking the contributors to raise the backport PRs which some repos do, GRPC comes to mind.)